### PR TITLE
Styled disclaimer component.  Additional FAQ for wrapped assets.

### PR DESCRIPTION
**Pull request by KPrasch**

### CODE REVIEW

File: packages/react-app/src/pages/manage/new/wrapping_faq.json [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
+    "answer": "No. Existing NU stakers *cannot* use the VendingMachine contract, only liquid NU / KEEP tokens can be wrapped to T. Instead existing NU/KEEP stakers will be grandfathered into Threshold via special staking adapters."
+  },
   {
     "question": "What are the mainnet conversion ratios for NU and KEEP into T?",
     "answer": "The ratio for 1 NU is 3.259242493160745 T.  The ratio for 1 KEEP is 4.783188631255016 T."
   },
   {
     "question": "Once I have converted tokens into T can I unwrap them?",
-    "answer": "Yes, unwrapping NU is supported by the VendingMachine contracts will also be supported by this interface in a near-future update."
+    "answer": "Yes, unwrapping NU is supported by the VendingMachine contracts will also be supported by this interface and the official threshold dashboard in a near-future update."
```
Comment (derekpierre) [inline packages/react-app/src/pages/manage/new/wrapping_faq.json:20]:
```suggestion
    "answer": "Yes, unwrapping NU / KEEP is supported by the VendingMachine contracts, and will be supported by this interface and the official Threshold dashboard in a near-future update."
```

File: packages/react-app/src/pages/manage/new/wrapping_faq.json [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
   {
     "question": "What happened to my NU staking details?",
     "answer": "The NuCypher Network staking contract was upgraded to ensure that legacy NU stakes can interoperate with the Threshold Network. Previous staking management functions are no longer supported through the NuCypher CLI/UI. Read-only stake information is still available and will be displayed here in a near-future update."
   },
+  {
+    "question": "Can I wrap staked NU/KEEP?",
+    "answer": "No. Existing NU stakers *cannot* use the VendingMachine contract, only liquid NU / KEEP tokens can be wrapped to T. Instead existing NU/KEEP stakers will be grandfathered into Threshold via special staking adapters."
+  },
   {
     "question": "What are the mainnet conversion ratios for NU and KEEP into T?",
```
Comment (derekpierre) [inline packages/react-app/src/pages/manage/new/wrapping_faq.json:15]:
```suggestion
    "question": "What are the conversion ratios for NU and KEEP into T?",
```

File: packages/react-app/src/pages/manage/new/wrapping_faq.json [hunk 3]
```diff
@@ -1,19 +1,23 @@
...
   },
   {
     "question": "What happened to my NU staking details?",
     "answer": "The NuCypher Network staking contract was upgraded to ensure that legacy NU stakes can interoperate with the Threshold Network. Previous staking management functions are no longer supported through the NuCypher CLI/UI. Read-only stake information is still available and will be displayed here in a near-future update."
   },
+  {
+    "question": "Can I wrap staked NU/KEEP?",
+    "answer": "No. Existing NU stakers *cannot* use the VendingMachine contract, only liquid NU / KEEP tokens can be wrapped to T. Instead existing NU/KEEP stakers will be grandfathered into Threshold via special staking adapters."
```
Comment (derekpierre) [inline packages/react-app/src/pages/manage/new/wrapping_faq.json:12]:
```suggestion
    "answer": "No. Existing NU stakers *cannot* use the VendingMachine contract, only liquid NU / KEEP tokens can be wrapped to T. Instead, existing NU/KEEP stakers will be grandfathered into Threshold via special staking adapters."
```

File: packages/react-app/src/pages/manage/new/wrapping_faq.json [hunk 4]
```diff
@@ -1,19 +1,23 @@
 [
   {
     "question": "What is this?",
-    "answer": "This interfaces exposes two deployments of the VendingMachine contract for wrapping NU or KEEP into T tokens. The VendingMachine smart contract converts NU to T using a fixed conversion ratio that will be available indefinitely. Existing NU stakers *cannot* use the VendingMachine contract, instead existing NU/KEEP stakers will be grandfathered into Threshold via special staking adapters. It is not necessary to keep your worker node up until until the instructions for Threshold staking are shared."
+    "answer": "This interfaces exposes two deployments of the VendingMachine contract for wrapping NU or KEEP into T tokens. The VendingMachine smart contract wraps NU into T using a fixed conversion ratio that will be available indefinitely. Existing NU stakers *cannot* use the VendingMachine contract, only liquid NU / KEEP tokens can be wrapped to T. Instead existing NU/KEEP stakers will be grandfathered into Threshold via special staking adapters. It is not necessary to keep your worker node up until until the instructions for Threshold staking are shared."
```
Comment (derekpierre) [inline packages/react-app/src/pages/manage/new/wrapping_faq.json:4]:
```suggestion
    "answer": "This interface exposes the two deployments of the respective VendingMachine contracts for wrapping NU or KEEP into T tokens. The VendingMachine smart contracts wrap NU / KEEP into T using a fixed conversion ratio that will be available indefinitely. Existing NU / KEEP stakes *cannot* use the VendingMachine contract, and only liquid NU / KEEP tokens can be wrapped to T. Instead, existing NU/KEEP stakes will be grandfathered into Threshold via special staking adapters. It is not necessary to keep your worker node up until the instructions for Threshold staking are shared."
```

File: packages/react-app/src/components/manage/disclaimer.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+    return (
+        <InputBox id="disclaimer">
+            <svg width="20" height="20" fill="currentColor"
+                 className="bi bi-megaphone mr-4" viewBox="0 0 16 16">
+                <path
+                    d="M13 2.5a1.5 1.5 0 0 1 3 0v11a1.5 1.5 0 0 1-3 0v-.214c-2.162-1.241-4.49-1.843-6.912-2.083l.405 2.712A1 1 0 0 1 5.51 15.1h-.548a1 1 0 0 1-.916-.599l-1.85-3.49a68.14 68.14 0 0 0-.202-.003A2.014 2.014 0 0 1 0 9V7a2.02 2.02 0 0 1 1.992-2.013 74.663 74.663 0 0 0 2.483-.075c3.043-.154 6.148-.849 8.525-2.199V2.5zm1 0v11a.5.5 0 0 0 1 0v-11a.5.5 0 0 0-1 0zm-1 1.35c-2.344 1.205-5.209 1.842-8 2.033v4.233c.18.01.359.022.537.036 2.568.189 5.093.744 7.463 1.993V3.85zm-9 6.215v-4.13a95.09 95.09 0 0 1-1.992.052A1.02 1.02 0 0 0 1 7v2c0 .55.448 1.002 1.006 1.009A60.49 60.49 0 0 1 4 10.065zm-.657.975 1.609 3.037.01.024h.548l-.002-.014-.443-2.966a68.019 68.019 0 0 0-1.722-.082z"/>
+            </svg>
+            <small>
+                This is a temporary wrapping UI for for early users/testers to upgrade their NU and KEEP to T
```
Comment (derekpierre) [inline packages/react-app/src/components/manage/disclaimer.js:15]: 👍

File: packages/react-app/src/pages/manage/new/wrapping_faq.json (hunk 1 above)
Comment (KPrasch) [reply packages/react-app/src/pages/manage/new/wrapping_faq.json:20]: Were these suggestions committed before merge, or did we lose them?

Comment (derekpierre) [reply packages/react-app/src/pages/manage/new/wrapping_faq.json:20]: I don't believe they were committed.

Comment (derekpierre) [reply packages/react-app/src/pages/manage/new/wrapping_faq.json:20]: Follow-up done in #161.

**Pull request merged**